docs/src/main/sphinx/connector/sql-update-limitation.fragment

```diff
@@ -0,0 +1,29 @@
+### UPDATE
+
+Only ``UPDATE`` statements with constant assignments and predicates are
+supported. For example, the following statement is supported:
+
+```sql
+UPDATE table SET col1 = 1 WHERE col3 = 1
+```
+
+The following statement is not supported because arithmetic expressions cannot
+be used with the ``SET`` command:
+
+```sql
+UPDATE table SET col1 = col2 + 2 WHERE col3 = 1
+```
+
+The following statement is not supported because ``AND`` and ``OR`` operators
```

Comment:
What about other logic operators like `NOT` ? Or mathematical operators?

Reply:
Supported predicates :

=, !=, >, <, >=, <=, IN, NOT IN